client/overview/index.js

```diff
@@ -11,25 +11,26 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies.
  */
-import Page from 'components/page';
-import { TestModeNotice } from 'components/test-mode-notice';
-import AccountStatus from 'components/account-status';
-import Welcome from 'components/welcome';
 import AccountBalances from 'components/account-balances';
-import DepositsOverview from 'components/deposits-overview';
+import AccountStatus from 'components/account-status';
 import ActiveLoanSummary from 'components/active-loan-summary';
+import ConnectionSuccessNotice from './connection-sucess-notice';
+import DepositsOverview from 'components/deposits-overview';
 import ErrorBoundary from 'components/error-boundary';
-import TaskList from './task-list';
-import { getTasks, taskSort } from './task-list/tasks';
+import FRTDiscoverabilityBanner from 'components/fraud-risk-tools-banner';
 import InboxNotifications from './inbox-notifications';
-import ConnectionSuccessNotice from './connection-sucess-notice';
-import ProgressiveOnboardingEligibilityModal from './modal/progressive-onboarding-eligibility';
 import JetpackIdcNotice from 'components/jetpack-idc-notice';
-import FRTDiscoverabilityBanner from 'components/fraud-risk-tools-banner';
-import { useDisputes, useGetSettings, useSettings } from 'wcpay/data';
+import Page from 'components/page';
+import PaymentActivity from 'wcpay/components/payment-activity';
+import ProgressiveOnboardingEligibilityModal from './modal/progressive-onboarding-eligibility';
+import SetupLivePaymentsModal from './modal/setup-live-payments';
 import strings from './strings';
+import TaskList from './task-list';
+import Welcome from 'components/welcome';
+import { getTasks, taskSort } from './task-list/tasks';
+import { TestModeNotice } from 'components/test-mode-notice';
```

Comment:
Nitpick: Maybe we can club all the `component` imports together.

Reply:
That's a good suggestion! Logical grouping is much better than mere alphabetical, which I had done earlier. 

https://github.com/Automattic/woocommerce-payments/pull/8412/commits/2ccf118e73e8bb22a4caf7c465cb2a3570fbf3ef